### ISSUE TITLE
Pretrained weights dimension mismatch

### ISSUE THREAD

**Issue by mkocabas** [tstandley/Xception-PyTorch#3]
Hi @tstandley,

Thanks for the great implementation. I tried to use the pretrained weights but it gives me shape mismatch error for each `conv` layer. Below is a sample output for one layer. My pytorch version is 0.4.
 
`While copying the parameter named "block1.rep.0.conv1.weight", whose dimensions in the model are torch.Size([64, 1, 3, 3]) and whose dimensions in the checkpoint are torch.Size([64, 3, 3]).
`

**Comment by blueardour** [comment]
@mkocabas  Hi, is there any progress on the issue? And have you test the accuracy of the model? thanks

**Comment by tstandley** [comment]
Can one of you check this out to see if it's fixed?

**Comment by tstandley** [comment]
@blueardour @mkocabas I think I fixed it. Let me know if you still have this issue.
